### PR TITLE
#3: 94-[TDD] Implement a method that can validates if the expression containing brackets is valid or not.

### PR DESCRIPTION
**Pull request by Maryam313**
Plz review the code..

The files are:
1)- MatchPair.class
2)- MatchPairTest.class

### CODE REVIEW

File: src/main/java/TDD2/MatchPair.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
+package TDD2;
+
+import java.util.Stack;
+
+import javax.sound.midi.VoiceStatus;
+
```
Comment (AnwaarAlbusaidi) [inline src/main/java/TDD2/MatchPair.java:5]: remove spaces

**Review by AnwaarAlbusaidi — APPROVED** [review]
well done

File: src/main/java/TDD2/MatchPair.java (hunk 1 above)
Comment (Maryam313) [reply src/main/java/TDD2/MatchPair.java:5]: done!

**Review by BaraahAlKalbani — APPROVED** [review]
great Work 👍🏻

**Review by FatemaAlbalushi — APPROVED** [review]
Great Work.

**Review by FatemaAlbalushi — APPROVED** [review]
Great Work

**Pull request closed without merging**